src/AppBundle/Controller/DonationController.php

```diff
@@ -116,11 +116,14 @@ public function resultAction(Request $request, Donation $donation)
 
         $retryUrl = $this->generateUrl('donation_informations', $parameters);
 
+        $membershipUtils = $this->get('app.membership_utils');
```

Comment:
Is a variable really needed here? `$this->get('app.membership_utils')->isInSubscriptionProcess()` seems fine to me.